src/main/java/io/rpg/config/model/GameWorldConfig.java

```diff
@@ -53,6 +55,17 @@ private GameWorldConfig() {
   @SerializedName(value = "rootLocation", alternate = {"root-location"})
   private String rootLocation;
 
+  /**
+   * Specifies the assets for each popup.
+   */
+  private String quizPopupBackground;
+  private String textPopupButton;
+  private String textImagePopupBackground;
+  private String textImagePopupButton;
+  private String textPopupBackground;
+  private String inventoryPopupBackground;
+
+
```

Comment:
As these fileds are required, I think it would be nice to update the [configuration docs](docs/configuration.md).